drafts/use-cases/3.2.pagination.md

```diff
@@ -18,13 +18,13 @@ So I can start reviewing them with option of loading more on demand.
 
 ```javascript
 const client = new HydraClient();
-let data = client.get("/api/events");
-for (const member of data.members) {
+let collection = client.get("/api/events");
+for (const member of collection.members) {
     // do something with the _member_, i.e. display it
 }
 // load some more on demand
-data = client.get(data.next);
-for (const member of data.members) {
+collection = client.get(data.view.next);
```

Comment:
@alien-mcl Maybe this snippet could already show the new API proposed in HydraCG/Heracles.ts#34